src/retasc/product_pages_api.py

```diff
@@ -0,0 +1,140 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+"""
+Production Pages API
+"""
+
+from functools import lru_cache
+
+from retasc import constants, requests_session
+
+
+class ProductPagesApi:
+    """
+    Product Pages API Client
+    """
+
+    def __init__(self, api_url: str):
+        self.api_url = api_url
+        self.session = requests_session.requests_session()
+
+    @lru_cache(maxsize=None)
+    @staticmethod
+    def releases_by_product_phase(
+        self,
+        product_shortname: str,
+        active_only: bool = True,
+        fields: str = "shortname,product_shortname,ga_date,phase_display",
+    ) -> list[dict]:
+        """
+        https://{pp_release_url}?product__shortname={product_shortname}&phase__lt=Unsupported&fields=a,b,c
+        The response have the following fields:shortname,product_shortname,ga_date,phase_display
+        phase id 1000 is unsupported, `lt` is less than
+        """
+        pp_release_url = self.api_url + "releases/"
```

Comment:
This assumes that `api_url` has trailing `/`. Maybe create a helper method:

```python
    def api_url_releases(self, id: int | None = None):
        return f"{self.api_url.rstrip('/')}/releases/{id or ''}"
```

Reply:
thanks, will update it